ee/vellum_cli/pull.py

```diff
@@ -146,12 +147,11 @@ def pull_command(
         request_options={"additional_query_parameters": query_parameters},
     )
 
-    zip_bytes = b"".join(response)
-    zip_buffer = io.BytesIO(zip_bytes)
-
-    error_content = ""
```

Comment:
[nit] we prefer to keep our try/except's targeted. we should try to just wrap this `join(response)` line